Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/service/decorator/cxf/TLSClientParametersFactory.java

```diff
@@ -64,36 +66,48 @@ private TLSClientParametersFactory() {
             trustFactory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
             trustFactory.init(cm.getTrustStore());
         } catch (UnrecoverableKeyException | KeyStoreException | NoSuchAlgorithmException e) {
-            LOG.error("Could not initialize key and trust managers: " + e.getLocalizedMessage(), e);
+            LOG.error("Could not initialize key and trust managers: {} ", e.getLocalizedMessage(), e);
         }
 
     }
 
     public static TLSClientParametersFactory getInstance() {
```

Comment:
Consider using a Singleton pattern here.  For an example within the CONNECT codebase, see X12EntityExceptionBuilder.java.